src/Integration.UnitTests/Binding/ProjectSystemFilterTests.cs

```diff
@@ -0,0 +1,171 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ProjectSystemFilterTests.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarLint.VisualStudio.Integration.Binding;
+using System;
+using System.Text.RegularExpressions;
+using System.Windows.Threading;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests.Binding
+{
+    [TestClass]
+    public class ProjectSystemFilterTests
+    {
+        private ConfigurableServiceProvider serviceProvider;
+        private ConfigurableVsProjectSystemHelper projectSystem;
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            this.serviceProvider = new ConfigurableServiceProvider();
+            this.projectSystem = new ConfigurableVsProjectSystemHelper(this.serviceProvider);
+            this.serviceProvider.RegisterService(typeof(IProjectSystemHelper), this.projectSystem);
+        }
+
+        #region Tests 
+
+        [TestMethod]
+        public void ProjectSystemFilter_IsAccepted_UnsupportedLanguage_IsFalse()
+        {
+            // Setup
+            var testSubject = this.CreateTestSubject();
+
+            var project = new ProjectMock("unsupported.vbproj");
+            project.SetVBProjectKind();
+
+            // Act
+            var result = testSubject.IsAccepted(project);
+
+            // Verify
+            Assert.IsFalse(result, "Project of unsupported language type should not be accepted");
+        }
+
+        [TestMethod]
+        public void ProjectSystemFilter_IsAccepted_SonarExcludeProperty_ReturnsPropertyValue()
+        {
+            // Setup
+            var testSubject = this.CreateTestSubject();
+
+            var project = new ProjectMock("supported.csproj");
+            project.SetCSProjectKind();
+
+            // Test case 1: property true -> not accepted
+            // Setup
+            project.SetBuildProperty(Constants.SonarQubeExcludeBuildPropertyKey, "true");
+            
+            // Act
+            var result = testSubject.IsAccepted(project);
+
+            // Verify
+            Assert.IsFalse(result, "Project with SonarExcludeProperty=false should NOT be accepted");
+
+            // Test case 2: property false -> is accepted
+            // Setup
+            project.SetBuildProperty(Constants.SonarQubeExcludeBuildPropertyKey, "false");
```

Comment:
What if the value is empty we gracefully treat is as false? Do we want to test this?